02_classify_stratified/nbconverted/plot_purity_regression_results.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Plot tumor purity prediction results
+# 
+# Here, we'll visualize the results of our tumor purity prediction. We're predicting real-valued purity labels here (e.g. elastic net regression).
+# 
+# Results analyzed here are generated by the `run_purity_prediction.py` script, without the `--classify` argument.
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib; matplotlib.rcParams['figure.dpi'] = 300
+import seaborn as sns
+
+import mpmp.config as cfg
+import mpmp.utilities.analysis_utilities as au
+
+
+# In[2]:
+
+
+# set results directory
```

Comment:
This is very minor, but usually, I structure my notebooks with section titles, such as "Setup" in this case, "Plot this and that", etc.

Reply:
Good idea! I'll start doing this more consistently.